adal/src/main/java/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -226,7 +225,7 @@ public String getSharedPrefPackageName() {
     /**
      * Gets expiration buffer.
      * 
-     * @return
+     * @return int
```

Comment:
this return javadoc doesn't provide any value. Either give more detailed info or remove it.